test/support/pages/alert.html

```diff
@@ -1,21 +0,0 @@
-<!DOCTYPE HTML>
-<html>
-  <head>
-    <title>Alert Test</title>
-  </head>
-  <body>
-    <h1>Alert Test</h1>
-    <a id='alert' href=# onclick="alert('this is an alert')")>Alert</a>
-    <a id='prompt' href=# onclick="doPrompt()">Prompt</a>
-    <p id='result'></p>
-    <script>
-    function doPrompt() {
-      var name = prompt("name", "");
-        if (name != null) {
-            document.getElementById("result").innerHTML =
-            "Hello " + name;
-        }
-    }
-</script>
-  </body>
-</html>
```

Comment:
I felt free to rename and modify this fixture, since it wasn't actually used anywhere. Let me know if you're somehow attached to it or have other reasons to keep this as is 😉